docs/kyma/10-02-basic-troubleshooting.md

```diff
@@ -54,3 +54,18 @@ minikube ssh -- docker run -i --rm --privileged --pid=host debian nsenter -t 1 -
 
 If you restart Kyma using unauthorized methods, such as triggering the installation when a Minikube cluster with Kyma is already running, the cluster might become unresponsive which can be fixed by reinstalling Kyma.
 To prevent such behavior, [stop and restart Kyma](#installation-install-kyma-locally-stop-and-restart-kyma-without-reinstalling) using only the method described.
+
+## Unable to deprovision Gardener cluster
```

Comment:
```suggestion
## Can't deprovision Gardener cluster
```
I know I suggested `unable` in the first place, but you were right, it's better to be consistent with the rest of the document.